src/index-debug.html

```diff
@@ -26,6 +26,9 @@
     <meta charset="utf-8">
     <title>Photoshop Design Space (development build)</title>
     <link rel="stylesheet" type="text/css" href="style.css" />
+    <script type="text/javascript">
+        var loadTime = new Date();
```

Comment:
Looks like we can use Chrome's `performance.timing` object to retrieve useful information like the time when browser starts loading the page (https://developer.mozilla.org/en-US/docs/Web/API/PerformanceTiming). With that, we no longer need the record the load time ourself.